frontend/src/Commons/Forms/InputModules/Summary/Summary.tsx

```diff
@@ -12,42 +12,66 @@ import InputModuleProps from 'src/Interface/InputModuleProps';
 import InputModuleInterface from 'src/Interface/InputModuleinterface';
 import { faList } from '@fortawesome/free-solid-svg-icons';
 import CustomLink from '@Commons/CustomLink/CustomLink';
-import { StyledFarmInfo } from './Summary.style';
+import { StyledFarmInfo, StyledContainer } from './Summary.style';
 
 const SummaryComponent: React.FC<InputModuleProps> = ({ farmDetails }) => (
-  <>
+  <StyledContainer>
     <StyledFarmInfo>
-      <p>
-        Farm Name:&nbsp;
-        {farmDetails.FarmName}
-      </p>
-      <p>
-        Year:&nbsp;
-        {farmDetails.Year}
-      </p>
-      <p>
-        Region:&nbsp;
-        {farmDetails.FarmRegion}
-      </p>
-      <p>
-        Berries:&nbsp;
-        {farmDetails.HasBerries ? 'Yes' : 'No'}
-      </p>
-      <p>Fields:&nbsp;</p>
-      {farmDetails.Fields.map((field) => (
-        <div key={field.FieldName}>
-          <p>{field.FieldName}</p>
-          <p>{field.Area}</p>
-          <p>{field.Comments}</p>
-        </div>
-      ))}
+      <div>
+        <p>
+          <span className="label">Farm Name:&nbsp;</span>
+          {farmDetails.FarmName}
+        </p>
+        <p>
+          <span className="label">Year:&nbsp;</span>
+          {farmDetails.Year}
+        </p>
+        <p>
+          <span className="label">Region:&nbsp;</span>
+          {farmDetails.FarmRegion}
+        </p>
+        <p>
+          <span className="label">Berries:&nbsp;</span>
+          {farmDetails.HasBerries ? 'Yes' : 'No'}
+        </p>
+        <span className="label">
+          <p>Fields:&nbsp;</p>
+        </span>
+        {farmDetails.Fields.map((field) => (
```

Comment:
use farmDetails.Fields.slice(1).map so that the initialDetails are removed from the map.

Reply:
Will fix this later. Does not work with loading .nmp files